frontend/src/components/configmap/List.tsx

```diff
@@ -1,13 +1,32 @@
 import { useTranslation } from 'react-i18next';
-import ConfigMap from '../../lib/k8s/configMap';
+import ConfigMap, { KubeConfigMap } from '../../lib/k8s/configMap';
+import { CreateResourceButton } from '../common/CreateResourceButton';
 import ResourceListView from '../common/Resource/ResourceListView';
 
+const BASE_EMPTY_CONFIG_MAP: KubeConfigMap = {
+  apiVersion: 'v1',
+  kind: 'ConfigMap',
+  metadata: {
+    creationTimestamp: '2023-04-27T20:31:27Z',
+    name: 'my-pvc',
+    namespace: 'default',
+    resourceVersion: '1234',
+    uid: 'abc-1234',
+  },
+  data: {},
+};
```

Comment:
I thought I had reviewed it a couple of weeks ago where I mentioned that the create timestamp should be set by us, nor the resource version or UID.
I had also mentioned that this base config should be at the cluster.ts's base object class, using a static var `baseObject`, and then every subclass should implement their own version in their class declaration (in ConfigMap.ts, etc.).

Reply:
these fields we should not set but are necessary to create these objects, would empty strings suffice? these will show up in the dialog as well and the user would have to delete them manually with the current setup

Reply:
I ran into a similar issue with the timestamp and the UID the fields would throw an error if I didn't include something in UID or timestamp. Maybe something like the solution from this PR to make those fields non required as input? 

https://github.com/headlamp-k8s/headlamp/pull/2155